cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -39,6 +42,69 @@
 ]
 
 
+@pytest.fixture()
+def ansible_linked_vm_action(
+    appliance, ansible_catalog_item, new_vm
+):
```

Comment:
I think we are not crossing 100 char limit

Reply:
Updated.